mantidimaging/gui/windows/main/view.py

```diff
@@ -474,13 +474,11 @@ def live_view_choose_directory(self) -> None:
             self.show_live_viewer(Path(live_data_directory))
 
     def show_live_viewer(self, live_data_path: Path) -> None:
-        if not self.live_viewer:
-            self.live_viewer = LiveViewerWindowView(self, live_data_path)
-            self.live_viewer.show()
-        else:
-            self.live_viewer.activateWindow()
-            self.live_viewer.raise_()
-            self.live_viewer.show()
+        live_viewer = LiveViewerWindowView(self, live_data_path)
+        self.live_viewer_list.append(live_viewer)
+        self.live_viewer_list[-1].activateWindow()
+        self.live_viewer_list[-1].raise_()
+        self.live_viewer_list[-1].show()
```

Comment:
I don't think we need these, as we no longer have the case of raising an existing live window